projects/canopy/src/lib/footer/footer.component.ts

```diff
@@ -19,33 +19,44 @@ export class LgFooterComponent {
   @HostBinding('class.lg-footer') class = true;
   @HostBinding('attr.role') role = 'contentinfo';
 
-  @Output() primaryLinkClicked = new EventEmitter<any>();
-  @Output() secondaryLinkClicked = new EventEmitter<any>();
+  @Output() primaryLinkClicked = new EventEmitter<Event>();
+  @Output() secondaryLinkClicked = new EventEmitter<Event>();
 
   private _logo: string;
-
   @Input()
   get logo(): string | null {
     return this._logo;
   }
-
   set logo(logo) {
     this._logo = logo;
     if (!this.logoAlt) {
       this.logoAlt = '';
     }
   }
 
+  private _secondLogo: string;
```

Comment:
I wonder if this should be `secondaryLogo` instead, to keep it consistent with the variable names for links? Bit pedantic I know.

Reply:
As discussed I will create a separate PR and put the change into `next` since it will be a breaking change because as the header also needs changing.